requirements.txt

```diff
@@ -31,6 +31,7 @@ pypet==0.6.1
 pyquaternion==0.9.9
 python-dateutil==2.8.2
 pytz==2022.7.1
+rich==13.3.2
```

Comment:
What is this used for? I didn't see it when skimming through just now

Reply:
It's for a loading bar and shows the progress as the CAD is built. It looks nice and was a dep. of a dep. anyway.

Reply:
Ya we can 👍🏼